src/routes/profile.js

```diff
@@ -84,7 +108,22 @@ router.post('/mod/ajax/processavatarrequest', isModMiddleware, (req, res) => {
             }
           });
       } else if (req.body.decision === false || req.body.decision === 'false') {
-        console.log(`search lower user: ${foundReq.forUsername.toLowerCase()}`);
+        const pattern = /pending_avatars\/.*$/;
+
+        if (
+          !foundReq.resLink.match(pattern) ||
+          !foundReq.resLink.match(pattern)
+        ) {
+          throw new Error(
```

Comment:
Let's pull this out to the top and validate their links together at the top.

Reply:
put it into the s3 relevant function instead as that is concerned with the overall link structure